doc/architecture/decisions/0034-use-devise-and-rotp-to-manage-authentication-with-sms-otp.md

```diff
@@ -0,0 +1,27 @@
+# 34. Use devise and rotp to manage authentication with sms OTP
+
+Date: 2022-01-25
+
+## Status
+
+Draft
+
+Supercedes [6. Use Auth0 for authentication](0006-use-auth0-for-authentication.md)
```

Comment:
Technically, it should be "supersedes", but enough people misspell it that it might become an alternative spelling in the future. ;)